packages/expo-cli/src/commands/eject/Github.ts

```diff
@@ -25,8 +26,8 @@ type RepoInfo = {
 };
 
 async function isUrlOk(url: string): Promise<boolean> {
-  const res = await got(url).catch(e => e);
-  return res.statusCode === 200;
+  const res = await fetch(url);
```

Comment:
The old `.catch(e => e)` would catch an eventual error and just make it the return value, causing the condition in the next line to fail, and the function would return `false` even in case of an exception. The new implementation doesn't do that, so in case of an exception, the function will throw an Unhandled Promise Rejection, bubbling all the way to a rather undescriptive error here: https://github.com/expo/expo-cli/blob/64eef52a6557cb66b9191819aa55ed96238a41b7/packages/expo-cli/src/commands/eject/createNativeProjectsFromTemplateAsync.ts#L148.